### ISSUE TITLE
Unable to turn fan on/off in HVAC

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2177]
```
FDS Version: 6.1.0
SVN Revision Number: 19412
Compile Date: Wed, 28 May 2014
Smokeview Version/Revision:
Operating System: Win 64-bit

Describe details of the issue below:

Hi guys,

I'm trying to control a fan assigned to a HVAC duct. I'd like to be able to turn it
on and off using a CTRL or a DEVC. The problem is that the fan does not respond to
CTRL's or DEVC's.

The simplest example I have been able to make is that I have added the following line
to the HVAC example 'fan_test.fds':

&DEVC XYZ=1,0.5,1, QUANTITY='TIME', SETPOINT=30., ID='TIMER', INITIAL_STATE=.TRUE.
/

And modified line 29 to:

&HVAC ID='LEFT',TYPE_ID='FAN',MAX_FLOW=0.16,MAX_PRESSURE=10., DEVC_ID='TIMER'/

in order to apply the device.


I have tested the device, by adding this line:

&OBST XB=2,3,0,1,0,1, SURF_ID='INERT', COLOR='ORANGE', DEVC_ID='TIMER' /


Thus, the file reads:

&HEAD CHID='fan_test', TITLE='Test of Fans Connecting Two Sealed Compartments, SVN
$Revision: 13979 $' /

&MESH IJK=20,10,10, XB=-3.0, 0.0,-1.0, 1.0, 0.0, 2.0 /
&MESH IJK=20,10,10, XB= 0.0, 3.0,-1.0, 1.0, 0.0, 2.0 /

&TIME T_END=60.0 /
&DUMP NFRAMES=60./
&MISC STRATIFICATION=.FALSE,Y_CO2_INFTY=0.,HUMIDITY=0./

&ZONE XB=-3.0, 0.0,-1.0, 1.0, 0.0, 2.0 /  Pressure Zone 1
&ZONE XB= 0.0, 3.0,-1.0, 1.0, 0.0, 2.0 /  Pressure Zone 2

&OBST XB= -0.15, 0.15,-1.0, 1.0, 0.0, 2.0 / Partition Wall


&VENT XB=-0.15,-0.15,-0.2, 0.2, 0.4, 0.6, COLOR='RED', ID='BLOW LEFT 1',SURF_ID='HVAC',IOR=-1/

&VENT XB= 0.15, 0.15,-0.2, 0.2, 0.4, 0.6, COLOR='GREEN', ID='BLOW LEFT 2',SURF_ID='HVAC',IOR=1/

&VENT XB=-0.15,-0.15,-0.2, 0.2, 1.4, 1.6, COLOR='BLUE', ID='BLOW RIGHT 1',SURF_ID='HVAC',IOR=-1/

&VENT XB= 0.15, 0.15,-0.2, 0.2, 1.4, 1.6, COLOR='YELLOW', ID='BLOW RIGHT 2',SURF_ID='HVAC',IOR=1/


&HVAC ID='LEFT 1',TYPE_ID='NODE',DUCT_ID='LEFT',VENT_ID='BLOW LEFT 1'/
&HVAC ID='LEFT 2',TYPE_ID='NODE',DUCT_ID='LEFT',VENT_ID='BLOW LEFT 2'/
&HVAC ID='RIGHT 1',TYPE_ID='NODE',DUCT_ID='RIGHT',VENT_ID='BLOW RIGHT 1'/
&HVAC ID='RIGHT 2',TYPE_ID='NODE',DUCT_ID='RIGHT',VENT_ID='BLOW RIGHT 2'/

&HVAC ID='LEFT',TYPE_ID='DUCT',NODE_ID='LEFT 1','LEFT 2',LENGTH=1,AREA=0.04,FAN_ID='LEFT',REVERSE=.TRUE.,LOSS=0,0/
&HVAC ID='RIGHT',TYPE_ID='DUCT',NODE_ID='RIGHT 1','RIGHT 2',LENGTH=1,AREA=0.04,LOSS=10,10/

&HVAC ID='LEFT',TYPE_ID='FAN',MAX_FLOW=0.16,MAX_PRESSURE=10., DEVC_ID='TIMER'/ This
line has been modified

&SLCF PBY=0.0,QUANTITY='TEMPERATURE',VECTOR=.TRUE. /
&SLCF PBY=0.0,QUANTITY='DENSITY' /
&SLCF PBY=0.0,QUANTITY='PRESSURE' /

&DEVC ID='pres_1', XYZ=-1.5, 0.0, 0.5,  QUANTITY='PRESSURE' /
&DEVC ID='pres_2', XYZ= 1.5, 0.0, 0.5,  QUANTITY='PRESSURE' /
&DEVC ID='vflow1', DUCT_ID='LEFT', QUANTITY='DUCT VOLUME FLOW' / 
&DEVC ID='vflow2', DUCT_ID='RIGHT', QUANTITY='DUCT VOLUME FLOW' / 


//-- Addition --
&DEVC XYZ=1,0.5,1,QUANTITY='TIME',SETPOINT=30. ,ID='TIMER', INITIAL_STATE=.TRUE. /
&OBST XB=2,3,0,1,0,1, SURF_ID='INERT', COLOR='ORANGE', DEVC_ID='TIMER' /


&TAIL /

All best
```

Original issue reported on code.google.com by `henrikhindborg` on 2014-08-06 15:20:40

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2014-08-06 15:30:05

**Comment by gforney** [comment]
```
This is a duplicate of the resolved issue #2213.

Put the DEVC_ID that controls the fan on the duct line that contains the fan.


```

Original issue reported on code.google.com by `drjfloyd` on 2014-08-06 15:32:49
